mocca-client/src/test/java/com/paypal/mocca/client/WireMockProvider.java

```diff
@@ -171,6 +171,9 @@ private static void configureMutationStubs() {
         final String EXPECTED_NULL_PARAMETER_REQUEST = "{\"query\":\"mutation{addSample(foo: \\\"zoo\\\", bar: \\\"\\\") {bar foo}}\"}";
         final String NULL_PARAMETER_RESULT = "{\"data\": {\"addSample\": {\"foo\": \"zoo\",\"bar\": \"\"}}}";
 
+        final String EXPECTED_ENUM_REQUEST = "{\"query\":\"query{addEnum(\\\"Sample1\\\")}\"}";
```

Comment:
Why did you remove the parameter name from here? How is the client supposed to know which parameter is which if there are multiple ones? 
This line was not supposed to be changed, only the response one.

Please add it back.